### DISCUSSION TITLE
Bug when extracting keys with possibly same tags, such as amenity=school OR building=school

### DISCUSSION THREAD

**Discussion by jkaucic** [ropensci/osmextract#284] [Q&A] [answered]
** Edit: I tried with smaller region and there the bug no longer exists. I'm trying to reproduce it again and come back. Sorry for the hassle!

**Comment by Robinlovelace** [comment]
Do you have a reproducible example? That will help.

**Reply by jkaucic** [reply, marked as answer]
Sorry, I reran the code and the issue no longer appears. It seems I had a hickup in my code. This q can be deleted!
